features/facebookImport/src/facebookImporter.jsx

```diff
@@ -28,10 +29,9 @@ import Overview from "./views/overview/overview.jsx";
 import ImportView from "./views/import/import.jsx";
 import ExploreView from "./views/explore/explore.jsx";
 import ExploreDetails from "./views/explore/details.jsx";
-import Loading from "./components/loading/loading.jsx";
+import ReportWrapper from "./views/report/reportWrapper.jsx";
```

Comment:
OK, this was a bit confusing, but it probably makes sense to put all imports together, and before the css import.

Reply:
Yeah when you let it auto import it will go below the css import. 
My idea when setting it up was 
1. React and react stuff (context)
2. components
3. CSS

Reply:
Yep, the confusing part was the direct substitution of "Loading" by "ReportWrapper"... But it's perfect the way it's now.